lib/index.js

```diff
@@ -50,32 +71,13 @@ Index.prototype.run = function() {
 
 
 Index.prototype.create = function() {
-  var isCoffee = !!require.extensions['.coffee'];
-
-  var template = (isCoffee ? [
-    ''
-    , 'exports.up = (next) ->'
-    , '  next()'
-    , ''
-    , 'exports.down = (next) ->'
-    , '  next()'
-    , ''
-  ] : [
-      ''
-    , 'exports.up = function(next) {'
-    , '  next();'
-    , '};'
-    , ''
-    , 'exports.down = function(next) {'
-    , '  next();'
-    , '};'
-    , ''
-  ]).join('\n');
+  var supportedExtensions = Object.keys(templates);
+  var extension = supportedExtensions.includes(cli['extension']) ? cli['extension'] : 'coffee';
```

Comment:
not sure if we want default extension to be `js` or `coffee` here.

Reply:
js